dotCMS/src/main/java/com/dotcms/rest/api/v1/container/ContainerResource.java

```diff
@@ -1413,4 +1429,81 @@ public final Response bulkUnarchive(@Context final HttpServletRequest  request,
                         new BulkResultView(unarchivedContainersCount,0L,failedToUnarchive)))
                 .build();
     }
-}
+
+    /**
+     * Saves a new working version of a container.
+     *
+     * @param request
+     * @param response
+     * @param containerForm
+     * @return
+     * @throws DotDataException
+     * @throws DotSecurityException
+     */
+    @POST
+    @Path("_add")
+    @JSONP
+    @NoCache
+    @Produces({MediaType.APPLICATION_JSON, "application/javascript"})
+    public final Response add(@Context final HttpServletRequest  request,
+            @Context final HttpServletResponse response,
+            final ContainerForm containerForm) throws DotDataException, DotSecurityException {
+
+        final InitDataObject initData = new WebResource.InitBuilder(webResource)
+                .requestAndResponse(request, response).requiredBackendUser(true).rejectWhenNoUser(true).init();
+        final User user         = initData.getUser();
+        final Host host         = WebAPILocator.getHostWebAPI().getCurrentHostNoThrow(request);
+        final PageMode pageMode = PageMode.get(request);
+        Container container     = new Container();
+
+        Logger.debug(this,
+                () -> "Adding container. Request payload is : " + ContainerResourceHelper.getInstance().getObjectToJsonString(containerForm));
+
+        this.containerAPI.save(container, containerForm.getContainerStructures(), containerForm, host, user, pageMode.respectAnonPerms);
+        Logger.debug(this, ()-> "The container: " + container.getIdentifier() + " has been added");
+
+        ActivityLogger.logInfo(this.getClass(), "Add Container",
+                "User " + user.getPrimaryKey() + " added " + container.getTitle(), host.getHostname());
+
+        return Response.ok(new ResponseEntityView(new ContainerView(container))).build();
+    }
+
+    /**
+     * Returns working version along with containerStructures {@link com.dotmarketing.portlets.containers.model.Container} based on the id
+     *
+     * @param request
+     * @param httpResponse
+     * @param containerId container identifier to get the working version.
+     * @return Response
+     * @throws DotSecurityException
+     * @throws DotDataException
+     */
+    @GET
+    @Path("/details")
```

Comment:
this too GET container?containerId=123 should be ok

Reply:
I will address this soon - I am working on it right now @jdotcms

Reply:
code has been updated and the details endpoint removed after adding a support in live and working endpoints to conditionally fetch contentTypes :)